cmd/devstream/init.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/merico-dev/stream/internal/pkg/config"
+	"github.com/merico-dev/stream/internal/pkg/pluginmanager"
+)
+
+var initCMD = &cobra.Command{
```

Comment:
Note: we need an init CMD according to ticket description here: https://github.com/merico-dev/stream/issues/17